pkg/config/replica_config.go

```diff
@@ -37,9 +37,6 @@ var defaultReplicaConfig = &ReplicaConfig{
 		WorkerNum: 16,
 	},
 	Sink: &SinkConfig{},
-	Cyclic: &CyclicConfig{
```

Comment:
What happens when we start ticdc with configuration that contains this item?

Reply:
`func (c *ReplicaConfig) UnmarshalJSON(data []byte) error {`  will ignore this item